system-test/spanner.js

```diff
@@ -2198,6 +2201,305 @@ describe('Spanner', function() {
             });
           });
         });
+
+        describe.only('structs', function() {
+          it('should bind a simple struct', function(done) {
+            var query = {
+              sql: 'SELECT @structParam.userf, @p4',
+              params: {
+                structParam: Spanner.struct({
+                  userf: 'bob',
+                  threadf: Spanner.int(1),
+                }),
+                p4: Spanner.int(10),
+              },
+            };
+
+            database.run(query, function(err, rows) {
+              assert.ifError(err);
+
+              var row = rows[0].toJSON();
+              assert.strictEqual(row.userf, 'bob');
+
+              done();
+            });
+          });
+
+          it('should bind null structs', function(done) {
+            var query = {
+              sql: 'SELECT @structParam.userf',
```

Comment:
Perhaps change this test to `SELECT @structParam IS NULL` ?

Reply:
Done!